### ISSUE TITLE
MaxListenersExceededWarning: Possible EventEmitter memory leak detected

### ISSUE THREAD

**Issue by ifeltsweet** [serverless/serverless#11077]
### Are you certain it's a bug?

- [X] Yes, it looks like a bug

### Is the issue caused by a plugin?

- [X] It is not a plugin issue

### Are you using the latest version?

- [X] Yes, I'm using the latest version

### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Issue description

When using Serverless Framework Compose to deploy multiple services in parallel (in my case 11) I receive the following warning:

```
(node:1081) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 exit listeners added to [process]. Use emitter.setMaxListeners() to increase limit
```

I wonder if Serverless Framework should automatically call `emitter.setMaxListeners()` with the number of parallel services being deployed.

### Service configuration (serverless.yml) content

Using `serverless-webpack`.

### Command name and used flags

serverless deploy

### Command output

```shell
(node:1081) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 exit listeners added to [process]. Use emitter.setMaxListeners() to increase limit
```

### Environment information

```shell
Framework Core: 3.17.0 (local)
Plugin: 6.2.2
SDK: 4.3.2
```

**Comment by pgrzesik** [comment]
Hey @ifeltsweet - thanks a lot for reporting. The issue is a duplicate of https://github.com/serverless/compose/issues/114 - let's continue discussion there and I'll close this issue to keep things clean.

**Comment by paul-civitas** [comment]
Hi the issue you linked to 404s.
